geopandas/array.py

```diff
@@ -572,9 +567,6 @@ def has_z(self):
 
     @property
     def has_m(self):
-        if not SHAPELY_GE_21:
-            raise ImportError("'has_m' requires shapely>=2.1.")
```

Comment:
Something I am wondering: we nowhere actually check the installed shapely version at runtime, right? So if you would be running with shapely 2.0, you'll get an AttributeError or something here. But we assume this should not happen because of the dependency metadata in pyproject.toml?